indent/scala.vim

```diff
@@ -536,7 +542,7 @@ function! GetScalaIndent()
   if prevline =~ '^\s*\*/'
    \ || prevline =~ '*/\s*$'
     call scala#ConditionalConfirm("18")
-    let ind = ind - 1
+    let ind = ind - 1 - scaladoc_extra_ind
```

Comment:
I think these might read better if you just make the variable indicate the complete extra indent of the asterisk, as in:

```vim
if get(g:, 'scala_scaladoc_indent', 0)
  let star_indent = 2
else
  let star_indent = 1
end
```

and then

```vim
return ind + star_indent
```